src/main/java/org/broadinstitute/hellbender/tools/exome/ReCapSegCaller.java

```diff
@@ -1,92 +1,97 @@
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.broadinstitute.hellbender.utils.Utils;
 import org.broadinstitute.hellbender.utils.param.ParamUtils;
 
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.DoubleStream;
+import java.util.stream.Stream;
 
 /**
- * Simple caller that
- * 1) estimates the value of normalized coverage corresponding to copy neutral segments
- * This value is not necessarily equal to zero.  For example, a large amplification on one segment
- * depresses the proportional coverage of all other segments.
- * 2) estimates the variance of targets' normalized coverage within a segment, assuming that
- * this variance is globally shared by all segments
- * 3) calls each segment based on a p-value for the null hypothesis that its targets'
- * normalized coverages are drawn from a normal distribution defined by the mean from 1)
- * and the variance from 2)
+ * <p>This caller mimics the legacy ReCapSeg Caller that was originally implemented in ReCapSeg v1.4.5.0.</p>
  *
- * Note:  Assumes that the incoming target file is in log_2(CR)
+ * <p>There is a small difference.  The python code was using the same algorithm as intersectBed, which was causing it to drop
+ *  the first target of each segment in calculations of the copy neutral targets.  The code here (targets.targets(s)) does
+ *  not do this.  This difference in the two codebases can cause a slight difference in the T calculation.  Hence, the
+ *  results of this code and the python code will not be exactly the same, but will be
+ *  very close.  A fix (to make this code match the python) has been deemed unworthy of our time.</p>
  *
- * @author David Benjamin
  */
 public final class ReCapSegCaller {
-
-    public static String AMPLIFICATION_CALL = "+";
-    public static String DELETION_CALL = "-";
-    public static String NEUTRAL_CALL = "0";
-
-    // default number of standard deviations from mean required to call an amplification or deletion
-    public static final double DEFAULT_Z_SCORE_THRESHOLD = 2.0;
-
-    //bounds on log_2 coverage we take as non-neutral a priori
-    public static final double NON_NEUTRAL_THRESHOLD = 0.3;
+    private static final Logger logger = LogManager.getLogger(ReCapSegCaller.class);
+    public static String AMPLIFICATION_CALL = CnvCaller.AMPLIFICATION_CALL;
+    public static String DELETION_CALL = CnvCaller.DELETION_CALL;
+    public static String NEUTRAL_CALL = CnvCaller.NEUTRAL_CALL;
 
     //bounds on log_2 coverage for high-confidence neutral segments
     public static final double COPY_NEUTRAL_CUTOFF = 0.1;
 
     private ReCapSegCaller() {} // prevent instantiation
 
+    private static double calculateT(final TargetCollection<TargetCoverage> targets, final List<ModeledSegment> segments) {
+
+        //Get the segments that are likely copy neutral.
+        // Math.abs removed to mimic python...
```

Comment:
Yup ... the comment says it all...  I think this was a bug in the python.
No action.